### PR TITLE
Added favicon and organized folder structures

### PR DESCRIPTION
**Pull request by isNan909**
Added some folders structures and added favicon to the application. 

Zoom-mirror 17 38 44

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,120 @@
+#Liscense
+LISCENSE
+Liscense
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
```
Comment (sbimochan) [inline .gitignore]: This app doesn’t contain these all files. Please make a commit with gitignore file removed.

Comment (isNan909) [reply .gitignore]: Ok the file is removed

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
-# zoom-mirror
+<img src = "https://i.imgur.com/EKgdUZY.png" width = "700">
+
+# Zoom mirror
+Be confident before you set you video-cam ON.
```
Comment (sbimochan) [inline README.md]:
```suggestion
Be confident before you set your web-camera ON.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+
+# What it does
```
Comment (sbimochan) [inline README.md]:
```suggestion
# What it does not
```

File: index.html [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
+      </button>
+    </div>
+    <div class="camera_arrow">
+      <img src="./assets/images/on_the_camera.png" alt="turn camera on arrow" />
+    </div>
+    <div id="subtitle"></div>
+  </div>
+</body>
+<script src="./scripts/main.js"></script>
+
```
Comment (sbimochan) [inline index.html]:
```suggestion
```

File: index.html [hunk 5]
```diff
@@ -1,37 +1,46 @@
...
+</html>
```
Comment (sbimochan) [inline index.html]:
```suggestion
</html>

```

**Review by sbimochan — CHANGES_REQUESTED** [review]

**Pull request merged**